### DISCUSSION TITLE
Cannot use man command

### DISCUSSION THREAD

**Discussion by Kritiqual** [z-shell/zi#244] [Q&A] [answered]
It seems like ZI is setting up the MANPATH variable through `ZI[MAN_DIR]` or`$ZPFX/man`. I can't execute any `man` command, e.g: `man man` is not working. I tried: `unset MANPATH` and the command is working again. How do I disable this behavior?

**Comment by ss-o** [comment, marked as answer]
Hi @Kritiqual, the fix has been applied in https://github.com/z-shell/zi/pull/245. 

Steps to resolve the issue:

```shell
zi self-update
unset MANPATH
exec zsh -il
```

Please let me know if this solves the unwanted behavior, thanks! :dart:

**Reply by Kritiqual** [reply]
Thanks for the quick fix!
